src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callsLimit = 3;
+
+  return function(a) {
+    return function(b) {
+      if (callsLimit <= 0 && callsLimit % 2 === 0) {
+        callsLimit--;
```

Comment:
I don't think it is limit, I think it's counter calls

Reply:
If you don't mind, I'll leave everything as it is, because in my opinion it's the same call counting, which is limited